### PR TITLE
Set GITHUB_OWNER envvar so that github provider modules will work

### PR DESCRIPTION
**Pull request by nicksantamaria**
### Related PRs
https://github.com/dpc-sdp/sdp-terraform-github/pull/152

### CODE REVIEW

File: .github/workflows/tf_apply.yml [hunk 1]
```diff
@@ -14,21 +14,23 @@ on:
...
-  SUMOLOGIC_ACCESSKEY: ${{ secrets.SUMOLOGIC_ACCESSKEY }}
   ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
   ARM_CLIENT_ID: ${{ secrets.ARM_CLIENT_ID }}
   ARM_CLIENT_SECRET: ${{ secrets.ARM_CLIENT_SECRET }}
   ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: ${{ inputs.aws_default_region }}
   EC_API_KEY: ${{ secrets.EC_API_KEY }}
+  GITHUB_OWNER: ${{ vars.TF_GITHUB_OWNER }}
```
Comment (nicksantamaria) [inline .github/workflows/tf_apply.yml:25]: Should this be `vars` or `secrets` ?

File: .github/workflows/tf_plan.yml [hunk 2]
```diff
@@ -7,21 +7,28 @@ on:
...
   ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: ${{ inputs.aws_default_region }}
   EC_API_KEY: ${{ secrets.EC_API_KEY }}
+  GITHUB_OWNER: ${{ inputs.tf_github_owner }}
+  GITHUB_TOKEN: ${{ secrets.TF_GITHUB_TOKEN }}
+  SUMOLOGIC_ACCESSID: ${{ secrets.SUMOLOGIC_ACCESSID }}
+  SUMOLOGIC_ACCESSKEY: ${{ secrets.SUMOLOGIC_ACCESSKEY }}
+  TF_LOG: DEBUG
```
Comment (FleetAdmiralButter) [inline .github/workflows/tf_plan.yml:29]: Could this be disabled by default and toggled via an input?

Comment (GROwen) [reply .github/workflows/tf_plan.yml]: oh yeah good pickup. It's gone 💨

File: .github/workflows/tf_apply.yml (hunk 1 above)
Comment (GROwen) [reply .github/workflows/tf_apply.yml]: I missed carrying over the changes from tf_plan for this, fixed now thanks @nicksantamaria

**Review by FleetAdmiralButter — APPROVED** [review]

**Comment by nicksantamaria** [comment]
Its my PR so I can't approve it, but consider it approved.

**Pull request merged**